gui-libs/greetd/greetd-0.6.1.ebuild

```diff
@@ -0,0 +1,113 @@
+# Copyright 2017-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+CRATES="
+arc-swap-0.4.6
+async-trait-0.1.31
+bitflags-1.2.1
+bytes-0.5.4
+cc-1.0.53
+cfg-if-0.1.10
+enquote-1.0.3
+fnv-1.0.7
+fuchsia-zircon-0.3.3
+fuchsia-zircon-sys-0.3.3
+getopts-0.2.21
+iovec-0.1.4
+itoa-0.4.5
+kernel32-sys-0.2.2
+lazy_static-1.4.0
+libc-0.2.70
+log-0.4.8
+memchr-2.3.3
+mio-0.6.22
+mio-named-pipes-0.1.6
+mio-uds-0.6.8
+miow-0.2.1
+miow-0.3.3
+net2-0.2.34
+nix-0.17.0
+pam-sys-0.5.6
+pin-project-lite-0.1.5
+proc-macro2-1.0.12
+quote-1.0.5
+redox_syscall-0.1.56
+rpassword-4.0.5
+ryu-1.0.4
+serde-1.0.110
+serde_derive-1.0.110
+serde_json-1.0.53
+signal-hook-registry-1.2.0
+slab-0.4.2
+socket2-0.3.12
+syn-1.0.21
+thiserror-1.0.17
+thiserror-impl-1.0.17
+tokio-0.2.11
+tokio-macros-0.2.4
+unicode-width-0.1.7
+unicode-xid-0.2.0
+users-0.9.1
+void-1.0.2
+winapi-0.2.8
+winapi-0.3.8
+winapi-build-0.1.1
+winapi-i686-pc-windows-gnu-0.4.0
+winapi-x86_64-pc-windows-gnu-0.4.0
+ws2_32-sys-0.2.1
+"
+
+inherit cargo systemd eutils
+
+DESCRIPTION="ipc based login daemon"
+
+HOMEPAGE="https://git.sr.ht/~kennylevinsen/greetd/"
+SRC_URI="https://git.sr.ht/~kennylevinsen/greetd/archive/${PV}.tar.gz -> ${P}.tar.gz
+	$(cargo_crate_uris ${CRATES})
+"
+RESTRICT="mirror"
```

Comment:
`RESTRICT="mirror"` is mostly for overlays.
remove it, we allow and encourage crate mirroring.
like 80% of those crates already on the mirrors.

Reply:
done